client/components/popover/index.jsx

```diff
@@ -71,89 +71,101 @@ var Popover = React.createClass( {
 		return null;
 	},
 
-	_showOrHideTip: function( prevProps ) {
-		if ( ! this._tip ) {
-			this._tip = new Tip( '' );
-			this._tip.classname = this.props.className;
-			this._container = this._tip.inner;
+	_initTip: function() {
+		this._tip = new Tip( '' );
+		this._tip.classname = this.props.className;
+		this._container = this._tip.inner;
+	},
+
+	_renderTip: function() {
```

Comment:
Splitting the code into shorter methods has been very useful to understand better how everything works.